posixlib/src/main/scala/scala/scalanative/posix/sched.scala

```diff
@@ -41,6 +41,30 @@ object sched {
       cpuset: Ptr[cpu_set_t]
   ): CInt = extern
 
+  /** Not defined in POSIX standard, might lead to runtime errors */
+  @name("scalanative_sched_other")
+  def SCHED_OTHER: CInt = extern
+
+  /** Not defined in POSIX standard, might lead to runtime errors */
+  @name("scalanative_sched_fifo")
+  def SCHED_FIFO: CInt = extern
+
+  /** Not defined in POSIX standard, might lead to runtime errors */
+  @name("scalanative_sched_rr")
+  def SCHED_RR: CInt = extern
```

Comment:
This 3 are mentioned in the GNU Posix so the comment is incorrect: 
https://man7.org/linux/man-pages/man3/pthread_attr_setschedpolicy.3.html  

> The supported values for policy are SCHED_FIFO, SCHED_RR, and
       SCHED_OTHER, with the semantics described in [sched(7)](https://man7.org/linux/man-pages/man7/sched.7.html).

Reply:
Seems they are inside POSIX: https://pubs.opengroup.org/onlinepubs/9699919799/functions/pthread_getschedparam.html

Thus, it contains one more optional scheduler policy `SCHED_SPORADIC` which I also included.